### ISSUE TITLE
why I install enveriment followd by install.md failled? (windows10)

### ISSUE THREAD

**Issue by tzc1994** [freechipsproject/chisel-bootcamp#73]
I downloaded coursier at first, then run the follow prompt and I meet
`2.8:0.2.1 sh.almond:scala-kernel_2.12.8:0.2.1 --sources --default=true -- --install
Unrecognized argument: --sources
Unrecognized argument: --default=true`

what the correct prompt is ?

**Comment by grebe** [comment]
I haven't tried it on windows, but I would think that the arguments would be the same. 

I think the command should actually be
```
java -noverify -jar coursier bootstrap -r jitpack -i user -I user:sh.almond:scala-kernel-api_2.12.8:0.2.1 sh.almond:scala-kernel_2.12.8:0.2.1 --sources --default=true -o almond ./almond --install
```

but it looks like it isn't recognizing arguments correctly.

What do you get if you do `java -noverify -jar coursier -h` and `java -noverify -jar cousier bootstrap -h`?

**Comment by tzc1994** [comment]
It doesn't wrok

PS C:\WINDOWS\system32> cd C:\Users\28614\Desktop
PS C:\Users\28614\Desktop> java -noverify -jar coursier bootstrap -r jitpack -i user -I user:sh.almond:scala-kernel-api_
2.12.8:0.2.1 sh.almond:scala-kernel_2.12.8:0.2.1 --sources --default=true -o almond ./almond --install
Unrecognized argument: --install
PS C:\Users\28614\Desktop> java -noverify -jar coursier -h
Coursier 1.1.0-M9
Usage: coursier [options] [command] [command-options]

Available commands: bootstrap, fetch, launch, resolve, spark-submit

Type  coursier command --help  for help on an individual command
PS C:\Users\28614\Desktop> java -noverify -jar cousier bootstrap
Error: Unable to access jarfile cousier
PS C:\Users\28614\Desktop> java -noverify -jar cousier bootstrap -h
Error: Unable to access jarfile cousier

**Comment by grebe** [comment]
It looks like that's not the right way to run coursier b/c it isn't finding it- have you downloaded coursier? Is it in the Desktop folder? Can you look at the [faq here](https://get-coursier.io/docs/faq)?

**Comment by tzc1994** [comment]
yes, I have downloaded courisier in Desktop folder. Thanks for your reply

**Comment by grebe** [comment]
Oh, also the `./almond --install` part of the command is supposed to be a separate command. It looks like the first command is almost working. The idea is that coursier makes an installer file called `almond`, so you need to run the coursier command before you can run the installer.

**Comment by luoxinjie1146** [comment]
when I run `.\almond --install`, the cmd reports `Error: Illegal char <:> at index 2: /E:/scala/almond`. How can I solve it?

**Comment by banklam** [comment]
On Windows 10 Enterprise, running in Windows PowerShell with admin privileges  
`java -noverify -jar coursier launch -r jitpack -i user -I user:sh.almond:scala-kernel-api_2.12.8:0.2.1 sh.almond:scala-kernel_2.12.8:0.2.1 --sources --default=true -- --install`
fails with the "unrecognized argument" error message as described by @tzc1994 , using instead
`java -noverify -jar coursier bootstrap -r jitpack -i user -I user:sh.almond:scala-kernel-api_2.12.8:0.2.1 sh.almond:scala-kernel_2.12.8:0.2.1 --sources --default=true -o almond`
does build almond, but running
`./almond --install`
returns the error reported by @luoxinjie1146 .
Hence, updating chisel-bootcamp is dead in the water here.

**Comment by seldridge** [comment]
As a workaround for Windows, is the hosted Binder instance sufficient for your use case? 

https://mybinder.org/v2/gh/freechipsproject/chisel-bootcamp/master

**Comment by banklam** [comment]
Yes, that'll work for now. Thanks.

**Comment by banklam** [comment]
I discovered a solution that works on my Windows 10 Enterprise system within a Windows PowerShell launched with admin privileges. Change the almond version to 0.6.0 as follows:
`java -noverify -jar coursier bootstrap -r jitpack -i user -I user:sh.almond:scala-kernel-api_2.12.8:0.6.0 sh.almond:scala-kernel_2.12.8:0.6.0 --sources --default=true -o almond`
followed by:
`./almond --install`

**Comment by luoxinjie1146** [comment]
Yes. It works.
Thanks!

**Comment by sagark** [comment]
Update for bootcamp attendees 9/6/19:

`java -noverify -jar coursier bootstrap -r jitpack -i user -I user:sh.almond:scala-kernel-api_2.12.8:0.6.0 sh.almond:scala-kernel_2.12.8:0.6.0 --sources --default=true -o almond`


`almond --install`
